app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
You need to have the following Actions in your controller:

- [ ] Kindly add a NEW action. This will help you initialize a new user. You can use this [article](https://guides.rubyonrails.org/action_controller_overview.html) to learn more about this. ⬇️ 

![user_new](https://user-images.githubusercontent.com/32361963/110916398-cf50aa00-8329-11eb-9910-6cfc0beaabb6.png)

- [ ] Kindly add a create action. The create action will help you initialize a new user, persist the new user data into the database, and if that is successful redirect you to the  new_user_path else it should just render new. The method you defined above. You can learn more about this in this [article](https://guides.rubyonrails.org/action_controller_overview.html) ⬇️ 

![user_params](https://user-images.githubusercontent.com/32361963/110916989-6f0e3800-832a-11eb-91e7-9e31b1bbfa44.png)

![user_create](https://user-images.githubusercontent.com/32361963/110917050-82210800-832a-11eb-8d65-b5b04b60b09b.png)

- [ ] Kindly create the edit action. What the edit action does is to get a certain user by id given in the params hash. You can learn more about this using this [article] (https://guides.rubyonrails.org/action_controller_overview.html) ⬇️ 
![user_route2 (1)](https://user-images.githubusercontent.com/32361963/110917438-f78cd880-832a-11eb-8b04-e36b858654c9.png)

- [ ] Kindly add the update action. This will allow you to get the user you need to update using the find method and the params hash, then save the updated data to the database, and if it's successful, it will 
redirect_to edit_user_path(<USER_ID>) else it will render the edit action above. ⬇️ 

![user_params](https://user-images.githubusercontent.com/32361963/110917906-86015a00-832b-11eb-87c1-120d36f128da.png)

![user_update](https://user-images.githubusercontent.com/32361963/110917975-9a455700-832b-11eb-88ba-2636263ad62a.png)

- [ ] Kindly add a private method to handle your params. What this method will do is to safely pass on the values in the params hash in the required actions above.  ⬇️ 
![user_params (1)](https://user-images.githubusercontent.com/32361963/110918196-e5f80080-832b-11eb-8005-c917efc736a5.png)